include/nihstro/shader_bytecode.h

```diff
@@ -96,9 +98,8 @@ struct SourceRegister {
             reg.value = index + 0x10;
         else if (type == RegisterType::FloatUniform)
             reg.value = index + 0x20;
-        else {
-            // TODO: Should throw an exception or something.
-        }
+        else
+            reg.value = -1; // Should not happen
```

Comment:
Actually, I think there is two ways of handling this:

- 1: mimic the `GetRegisterType()` function choice on which these functions are based and treat `FloatUniform` as the default case.
It's probably the best option if we're confident `GetRegisterType` will never have to return other types than the ones it currently does.

- 2: write a new enum `SourceRegisterType` with a subset of `RegisterType` values and replace the `if`/`else` series by an exhaustive switch.
It may be the more flexible way of doing this, but we have to think of a way to make sure the two types remain perfectly associated in the future.

What are your thoughts?